Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM docker.io/rocker/r-ver:4.3.1
+
+RUN CRAN_LIKE_URL="https://packagemanager.posit.co/cran/__linux__/jammy/2024-04-05"; \
```

Comment:
Is this semi-colon needed or was it intentional?

Reply:
It is necessary!

Removing it causes the build to fail